src/seedu/addressbook/AddressBook.java

```diff
@@ -383,11 +412,28 @@ private static String executeCommand(String userInputString) {
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
+        case COMMAND_SORT_WORD:
+        	return executeSortInAlphabeticOrder();
         default:
             return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
         }
     }
-
+    
+    /* Sort in Alphabetical Order
```

Comment:
additionally, the comment style should use the active verb form **sorts** instead of **sort**